Decred-Never-Stops.md

```diff
@@ -0,0 +1,54 @@
+| Author | João Paulo Sant'Anna |
+| :---- | :---- |
+| Title | Decred never stops! |
```

Comment:
Need title ideas, to make this more engaging. Maybe ask @exitus, he also's has great ideas for titles.